### PR TITLE
Fixed views collection params

### PR DESCRIPTION
**Pull request by psav**
Params in yaml needed to be altered based on appliance stream. Basically
verpicking. If this kind of solution is accepted, maybe we can make this
a feature of the yaml loader, but for now, this seems to work well enough.

Tests will fail, because there is an associated yaml change. However, if I merge it, without this change, it will also break current tests. So I'll wait until the change it merged.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 4ed507ce628582753fae58bcbc1b007e6b32ae6b:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/configure/test_default_views.py [hunk 1]
```diff
@@ -7,14 +7,23 @@
...
 
+def minimise_dict(item):
+    if isinstance(item, dict):
+        return version.pick({str(k): v for k, v in item.iteritems()})
+    else:
+        return item
+
 try:
     gtl_params = cfme_data['defaultview_data']['gtl']['infra']
+    gtl_params = [minimise_dict(item) for item in gtl_params]
```
Comment (seandst) [inline cfme/tests/configure/test_default_views.py]: `gtl_params = map(minimise_dict, gtl_params)`

Also minimize, but I can let that one go. :)

**Comment by seandst** [comment]
I sorta like this as a start, but we should consider having a `VersionDict` type that generally acts like a dict, but does the version pick work automatically when `__getitem__` is hit.

**Comment by psav** [comment]
Yeh.....that was my thoughts as well, but if you're happy to let this one in for now, please merge and I will push the yaml update with another I'm working on in a bit

**Pull request merged**